recipe/meta.yaml

```diff
@@ -1,29 +1,61 @@
-# Switch the version of this recipe based on the VC_VERSION environment variable.
-{% set VERSION = os.environ.get('VC_VERSION', '9')|string %}
-{% set runtime_years = {'9': '2008', '10': '2010', '14': '2015'} %}
-{% set runtime_year = runtime_years[VERSION] %}
+# version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
+#     This version number follows the "platform toolset"
+{% set vcver="14" %}
+{% set vsyear="2015" %}
```

Comment:
Will `2017` be added once the `2015` is in sync with AnacondaRecipes?

Reply:
Nvm, missed the "and vs2017" in https://github.com/conda-forge/vc-feedstock/pull/10/files#r343789285